### PR TITLE
jsonp is called instead of json in cross domain requests

### PR DESCRIPTION
**Pull request by ubershmekel**
This was surprising to me and I'm still not sure how to handle cross-domain jsonp 404 responses.

### CODE REVIEW

**Comment by arthurvr** [comment]
@ubershmekel This PR needs some updates. See comments above. It would be nice if you could do them?

**Comment by ubershmekel** [comment]
Fixed the complaints and fixed the merge. From my testing `crossdomain` does not help detect the 404. The 404 being undetectable may be a security feature.

**Comment by arthurvr** [comment]
Thanks @ubershmekel. What do we think @gnarf @kswedberg?

**Comment by dmethvin** [comment]
I definitely like this formatting better. For this:

> Cross-domain requests may invoke "jsonp" unless explicitly "jsonp" is set to false.

I'd change it to something like:

> Cross-domain "json" requests are converted to "jsonp" unless the request includes `jsonp: false` in its request options.

**Comment by arthurvr** [comment]
ping @ubershmekel

**Comment by ubershmekel** [comment]
Sorry guys, I was abroad for a few weeks. It's a bit of a pain that every time I come back to this tiny change the pull request has merge conflicts.

I do like the proposed wording. If I fix it per @dmethvin would that be the final iteration?

**Comment by arthurvr** [comment]
> It's a bit of a pain that every time I come back to this tiny change the pull request has merge conflicts.

For now feel free to not care about merge conflicts. I can take care of them while merging.

**Comment by ubershmekel** [comment]
Merged with the new wording.

**Comment by ubershmekel** [comment]
Ping @dmethvin @arthurvr before the next conflict.

**Comment by dmethvin** [comment]
Whoa, sorry this got buried. LGTM!

File: entries/jQuery.ajax.xml [hunk 1]
```diff
@@ -74,8 +74,15 @@ $.ajax({
...
-          </li></ul>
+          <ul>
+            <li>"xml": Returns a XML document that can be processed via jQuery.</li>
```
Comment (arthurvr) [inline entries/jQuery.ajax.xml]: > "xml"

I wonder if these shouldn't be in `<code>` elements. For the rest this LGTM!

**Comment by arthurvr** [comment]
I'm going to make that trivial change myself and land this one. Thanks for your hard work, @ubershmekel , and sorry for it taking too long :)

**Pull request closed without merging**